File: pset6/mario.py
#import cs50 library to use get_int function
import cs50

#while loop used to accept an appropriate input
while (True):
    print("Please enter a height between 0 and 23: ", end='')
    x = cs50.get_int()
    if (x >= 0 and x <= 23):
        break

#for loop logic used to print appropriate number of spaces and hashes
for j in range(x):
    print(" " * (x - (j + 1)), end='')
    print("#" * (j + 1), end='')
    print("  ", end='')
    print("#" * (j + 1))

File: pset1/mario/less/mario.c
#include <stdio.h>
#include <cs50.h>

int main(void)
{

    /* Initialize variable */
    int x, i, n, j;

    /* Prompt user input for an integer between 0 and 23 inclusive */
    do
    {
        x = get_int("How many blocks do you want? ");
    }
    while (x < 0 || x > 23);


    /* Create a for loop to control the height of the pyramid */
    for (j = 0; j < x; j++)
    {

        /* Create a for loop to control the number of spaces */
        for (i = 0; (x - (i + 1)) > j; i++)
        {
            printf(" ");

        }

        /* Create a for loop to control the number of hashes */
        for (n = 0; (j + 2) > n ; n++)
        {
            printf("#");

        }

        /* Print a new line for each row of the pyramid */
        printf("\n");

    }
}


